### DISCUSSION TITLE
Config file in docker

### DISCUSSION THREAD

**Discussion by darrepac** [abraunegg/onedrive#1611] [Q&A] [answered]
Hi

When onedrive is running through docker and I make a change in the config file. Do I have to restart the program/docker or it looks at it before every monitor run?

thx

**Comment by abraunegg** [comment, marked as answer]
@darrepac 
After every config change you need to restart your Docker container to pick up the new configuration

**Reply by darrepac** [reply]
ok thanks

**Comment by darrepac** [comment]
Just tell me what I did wrong:
I put a "config" file into `/var/lib/docker/volumes/onedrive_conf`
In the config file, I just put 1 line:
`monitor_interval = "86400"`

I then restarted the container but it doesn't seem to take my monitor interval into account. Did I miss something?

**Reply by abraunegg** [reply]
@darrepac 
> I put a "config" `file into /var/lib/docker/volumes/onedrive_conf`

This is not the right way - as per the documentation: https://github.com/abraunegg/onedrive/blob/master/docs/Docker.md#6-edit-the-config

You need to create / edit the config file via:
```
docker volume inspect onedrive_conf
```
